### DISCUSSION TITLE
`ProgressBar` must set `MinWidth`?

### DISCUSSION THREAD

**Discussion by Coloryr** [AvaloniaUI/Avalonia#14305] [Q&A] [answered]
just watch the pic you know.
```
<ProgressBar
    Width="100"
    Height="8"
    MinWidth="100"
    Maximum="32767"
    Value="800" />
```
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/1025169b-6642-476f-9ad8-de89ae9606d8)
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/9b3bd42c-4f3a-4115-8698-391de8597fd3)
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/1844743a-fcde-4dc1-934b-2f8a927cfbed)
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/a9e0fbbb-f87f-4eb6-82e5-aa699367b9a6)

**Comment by timunie** [comment, marked as answer]
Check original style file, there is MinWidth set. So if you want to have it smaller, you can override this in you own style.
